lib/src/test/java/vmath/Mat2Test.java

```diff
@@ -22,6 +22,42 @@ void givenIndices_whenGetCalled_thenReturnValueMatches() {
         //TODO
     }
 
+    @Test
+    void givenArgs_whenMat2Created_thenMatrixHasGivenValues() {
+        float[] validValues = {
+                2.0f, 3.0f,
+                4.0f, 5.0f
+        };
+
+        Mat2 mat2 = new Mat2(validValues);
+
+        for (int i = 0; i < 2; i++) {
+            for (int j = 0; j < 2; j++) {
+                assertEquals(validValues[i * 2 + j], mat2.get(i, j));
+            }
+        }
+    }
+
+    @Test
+    void givenWrongSize_whenMat2Created_thenThrowsException() {
+        float[] invalidSize = {
+                2.0f, 3.0f,
+                4.0f
+        };
+
+        assertThrows(IllegalArgumentException.class, () -> new Mat2(invalidSize));
+    }
+
+    @Test
+    void givenNaNValues_whenMat2Created_thenThrowsException() {
+        float[] invalidValues = {
+                Float.NaN, Float.NaN,
+                Float.NaN, Float.NaN
+        };
+
+        assertThrows(IllegalArgumentException.class, () -> new Mat2(invalidValues));
```

Comment:
In the other two PRs approved before you, there were additional assertions to check when just one of the values is `NaN` (and not the first one).

Reply:
Indentation and variable names are again a problem in every test method.